eslint.config.js

```diff
@@ -0,0 +1,43 @@
+const enactConfig = require('eslint-config-enact/strict');
+const importPlugin = require('eslint-plugin-import');
+const prettierPlugin = require('eslint-plugin-prettier');
+const prettierRecommended = require('eslint-config-prettier');
+
+module.exports = [
+	{
+		languageOptions: {
+			ecmaVersion: 'latest',
+			sourceType: 'module',
+			parserOptions: {
+				ecmaFeatures: {
+					jsx: true
+				}
+			}
+		},
+		plugins: {
+			enactConfig,
+			import: importPlugin,
+			prettierPlugin,
+			prettierRecommended
+		},
+		rules: {
+			'react/forbid-foreign-prop-types': 'off', // proptypes not removed in storybook config
+			'import/no-unresolved': ['error', {commonjs: true, caseSensitive: true}],
+			'import/named': 'error',
+			'import/first': 'warn',
+			'import/no-duplicates': 'error',
+			'import/extensions': ['warn', 'always', {js: 'never', json: 'always'}],
+			'import/newline-after-import': 'warn',
+			'import/order': [
+				'warn',
+				{
+					'newlines-between': 'never',
+					groups: ['builtin', 'external', 'internal', 'parent', 'sibling', 'index']
+				}
+			],
+			'prettierPlugin/prettier': 'error',
+			'prettierRecommended/arrow-body-style': 'off',
+			'prettierRecommended/prefer-arrow-callback': 'off'
+		}
+	}
+];
```

Comment:
Could you explain the changed config file in more detail in "Resolution"?
I compared these configurations with the deleted .eslintrc.js, and there seem to be many changes.

Reply:
I removed the env option according to the docs: 
https://eslint.org/blog/2022/08/new-config-system-part-2/ ("For flat config, we decided to remove the env key completely.")
https://eslint.org/blog/2022/08/new-config-system-part-3/ ("The envs option has been removed.")

I was not sure how to handle "extends" property which is no longer used in the new config file. I first added the configs that were in the "extends" array in the "plugins" object, but now I pushed a change according to this: https://github.com/jsx-eslint/eslint-plugin-react#configuring-shared-settings.
We now get the following Travis error: Unexpected key "extends" found.

For prettier/recommended I added the rules manually like here: https://github.com/prettier/eslint-plugin-prettier#recommended-configuration. 

I will add this to "Resolution" after I manage to fix the issues above. Thank you!

Reply:
ok...hope the travis works...